pkg/processor/sumologicschemaprocessor/README.md

```diff
@@ -271,3 +278,21 @@ The result is:
   }
 }
 ```
+
+### Severity Attribute
+
+Some fields that log entries consist are not displayed as fields in Sumo Logic out of the box.
+It's possible to convert specific attributes to fields. List of currently supported attributes is below:
+
+`add_severity_number_attribute`
+`add_severity_text_attribute`
+`add_space_id_attribute`
+`add_trace_id_attribute`
+
+In order to report one of them as field, following configuration is needed:
+
+```yaml
+    add_severity_number_attribute:
+        enabled: true
+        attribute_name: "loglevel"
+```
```

Comment:
Can all of them be enabled or only one at one time? Maybe it's better to squash this into one attribute and disable it by selecting `none`, empty string or something like that?

Reply:
My idea was to be able to enable one by one, however I have not seen requirements regarding this from customer side.

Reply:
Ok, I've read the discussion above. In my opinion, the best thing would be to define something like this (the name should probably be different, I'm not sure how to call all these attributes):
```yaml
add_log_metadata_attributes:
  metadata: ["severity_no", "severity_text", "span_id", "trace_id"]
```

Reply:
ok, I can update implementation according to your suggestion, @astencel-sumo @sumo-drosiek What's your opinion on that? I would like to avoid yet another round of changes regarding configuration.

Reply:
I'm not convinced that squashing all configurations into a single property is any better than the current solution. We can discuss in a meeting.

Reply:
> I'm not convinced that squashing all configurations into a single property is any better than the current solution. We can discuss in a meeting.

same as we skip `attribute_name`